site/en/guide/tpu.ipynb

```diff
@@ -304,9 +309,9 @@
         "id": "mgUC6A-zCMEr"
       },
       "source": [
-        "### Train a model using Keras high level APIs\n",
+        "### Train the model using Keras high-level APIs\n",
         "\n",
-        "You can train a model simply with Keras fit/compile APIs. Nothing here is TPU specific, you would write the same code below if you had mutliple GPUs and where using a `MirroredStrategy` rather than a `TPUStrategy`. To learn more, check out the [Distributed training with Keras](https://www.tensorflow.org/tutorials/distribute/keras) tutorial."
+        "You can train your model with Keras `fit` and `compile` APIs. There is nothing TPU-specific in this step — you write the code as if you were using mutliple GPUs and a `MirroredStrategy` instead of the `TPUStrategy`. You can learn more in the [Distributed training with Keras](https://www.tensorflow.org/tutorials/distribute/keras) tutorial."
```

Comment:
```suggestion
        "You can train your model with Keras `fit` and `compile` APIs. There is nothing TPU-specific in this step—you write the code as if you were using mutliple GPUs and a `MirroredStrategy` instead of the `TPUStrategy`. You can learn more in the [Distributed training with Keras](https://www.tensorflow.org/tutorials/distribute/keras) tutorial."
```